src/picoprobe_config.h

```diff
@@ -48,9 +48,19 @@
 
 // PIO config
 #define PROBE_SM 0
+
+#ifndef USE_WS2812
 #define PROBE_PIN_OFFSET 2
-#define PROBE_PIN_SWCLK PROBE_PIN_OFFSET + 0 // 2
-#define PROBE_PIN_SWDIO PROBE_PIN_OFFSET + 1 // 3
+#else
+#define PROBE_PIN_OFFSET 22 // Use the QTPy's QT STEMMA
```

Comment:
Feels like this is kinda orthogonal to the `USE_WS2812` setting? (e.g. there might be some non-QTPy board that wants to use WS2812?)

Hmmm, I never noticed that https://learn.adafruit.com/adafruit-qt-py-2040/pinouts doesn't explicitly spell out which GPIO pins are used by the STEMMA connector - is that something that you could get added @dhalbert ?

Reply:
In the past we have supplied Arduino and CircuitPython pin names/numbers in our Learn Guides, and avoided supplying the "raw" chip pin numbers, because it tends to be confusing and increases support issues. The raw pin numbers are available in source, or more easily, on the schematic. GIven pico-sdk, though, we might reconsider this. I will ask @ladyada.

Reply:
Perhaps this would be less of an issue if https://github.com/raspberrypi/pico-sdk/blob/master/src/boards/include/boards/adafruit_qtpy_rp2040.h contained the STEMMA etc. pin numbers? :shrug:

Reply:
FYI, Stemma pins grabbed from this: https://blog.adafruit.com/2021/05/20/pin-reference-adafruit-qt-py-rp2040/ which is the only place I could find it (see first comment above)

Reply:
For now, the schematics are available on the download pages in the Learn Guides for each board: https://learn.adafruit.com/adafruit-qt-py-2040/downloads.

The pin mappings in CircuitPython, which correspond to the silkscreen mappings, are in the `pins.c` file for each board, e.g. https://github.com/adafruit/circuitpython/blob/main/ports/raspberrypi/boards/adafruit_qtpy_rp2040/pins.c.

I'm not sure why we haven't added that colorful pin diagram to the QT Py 2040 guide yet. I think that may be just an oversight.